theme/themepacks.less

```diff
@@ -14,6 +14,7 @@
     --pxt-page-background: white;
     --pxt-page-foreground: black;
     --pxt-page-foreground-light: #767676;
+    --pxt-page-foreground-lighter: #888888;

```

Comment:
Ugh. I need to revisit our theme vars soon! Great to capture this requirement though, and thank you for adding it as a var.

Reply:
For the criteria table, we use --pxt-content-accent. Would that color work for this use case as well?

Reply:
Yeah, I actually like that!! I'm just wondering if the header's bottom border should be darker, though. I played with it and I'm not sure between these:

**With header bottom border darker**
<img width="537" alt="Screenshot 2024-03-01 150057" src="https://github.com/microsoft/pxt/assets/49178322/e7122afd-e6af-481d-993c-08f1569e7335">

**With all bottom borders being the same**
<img width="534" alt="Screenshot 2024-03-01 150235" src="https://github.com/microsoft/pxt/assets/49178322/de4f598b-4c7a-4987-9bb8-f2b3b0ceffb1">

Reply:
2nd one (same borders) seems cleaner to me.